src/js/components/Ballot/BallotItem.jsx

```diff
@@ -21,9 +21,9 @@ export default class BallotItem extends Component {
   }
 
   render () {
-    return <div className="ballot-item well well-skinny gutter-top--small">
+    return <div className="ballot-item gutter-top--small">
```

Comment:
I am (somewhat aggressively) trying to remove as much 'visual nesting' of elements as possible. `.well` didn't seem like it was necessary, but I'm open to any push back.